src/client/actions/GuildMemberUpdate.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Action = require('./Action');
+const { Status, Events } = require('../../util/Constants');
+
+class GuildMemberUpdateAction extends Action {
```

Comment:
Any...specific reason this was turned into an action?

Reply:
To be able to use `Action#getMember` (on line 20)